### PR TITLE
features: Minor edits

### PR DESCRIPTION
**Pull request by prtksxna**

### CODE REVIEW

File: content/pages/features.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ sortorder: 20
 <div class="row">
   <div class="col-lg-6">
     <h2>Keep your data</h2>
-    All your data belong to you. You can download all your articles in many formats: PDF, ePUB, .mobi, JSON, CSV, txt or HTML. 
+    All your data belongs to you. You can download all your articles in many formats: PDF, ePUB, .mobi, JSON, CSV, txt or HTML.
```
Comment (nicosomb) [inline content/pages/features.md]: Data is a plural word, right? Ping @tcitworld

Comment (tcitworld) [inline content/pages/features.md]: It's *belongs* indeed.

File: content/pages/features.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ sortorder: 20
   </div>
   <div class="col-lg-6">
     <h2>Comfortable reading</h2>
-    wallabag extracts article's content (and only content!) and displays it in a comfortable view.<br />
+    wallabag extracts the article's content (and only content!) and displays it in a comfortable view.<br />
```
Comment (tcitworld) [inline content/pages/features.md]: and only it's content

**Comment by nicosomb** [comment]
Thank you!

**Pull request merged**